src/ListComponent.tsx

```diff
@@ -88,7 +98,7 @@ export const ListComponent = React.memo(function ListComponent({
         >
             {alignItemsAtEnd && <Reactive.View $style={() => ({ height: paddingTop$.get() })} />}
             {ListHeaderComponent && (
-                <Reactive.View $style={ListHeaderComponentStyle}>{ListHeaderComponent}</Reactive.View>
+                <Reactive.View $style={ListHeaderComponentStyle}>{getComponent(ListHeaderComponent)}</Reactive.View>
```

Comment:
note: the `getComponent` calls are not memoized and might have performance impact.